src/referencePackages/src/system.buffers/4.5.1/ref/netstandard1.1/System.Buffers.cs

```diff
@@ -4,7 +4,6 @@
 // ------------------------------------------------------------------------------
 // Changes to this file must follow the http://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
-[assembly: System.Reflection.AssemblyMetadata("source", "source-build-reference-packages")]
```

Comment:
I just realized that you must be running with an older toolset locally.  You should clean your repo to delete you local artifacts directory and .dotnet.  These attributes were added some time ago now.